src/server/httpServer.js

```diff
@@ -42,12 +42,7 @@ export default function server(store) {
           }
           res.end();
         } else {
-          const value = await store.call('read', query, {
-            ...options,
-            raw: true,
-          });
-          res.writeHead(200);
-          res.end(serialize(value));
+          return log('Get not supported for this type. Try POST');
```

Comment:
Logging and returning is not enough, you need to send a response to the client.

You can do this here by throwing an error, because there is a catch block below that will handle sending the error.

I suggest `throw Error('httpServer.get_unsupported')` to be consistent with Graffy's error codes.

Reply:
I did do this, then I got a warning : ``. On googling, I found we must not throw exceptions conditionally. [ref](https://stackoverflow.com/questions/729379/why-not-use-exceptions-as-regular-flow-of-control)

Reply:
What warning?